fakeserver/server.go

```diff
@@ -287,3 +345,33 @@ func generateSelfSignedCert() (tls.Certificate, error) {
 
 	return cert, nil
 }
+
+// Stub implementation for GetConfigurationUpdates
+func (s *server) GetConfigurationUpdates(stream pb.KubernetesInfoService_GetConfigurationUpdatesServer) error {
+	logger.Info("GetConfigurationUpdates stream started")
+	for {
```

Comment:
You have to read the messages received from the stream.
Right now, it's only keepalives on this stream.
https://github.com/illumio/cloud-operator/blob/df44b593931b44dd34b00bfa7b25eb248c9f90a5/api/illumio/cloud/k8sclustersync/v1/k8s_info.proto#L408-L412